x509/tests/certificate.rs

```diff
@@ -67,7 +67,9 @@ pub struct DeferDecodeTBSCertificate<'a> {
     /// Defer decoded field
     pub extensions: &'a [u8],
 }
-
+pub fn default_zero_u8() -> u8 {
+    0
+}
```

Comment:
Please don't. Use `Default::default`.

Reply:
As part of the realignment as noted above.